src/components/LeftSidebar/InvitationHandler.vue

```diff
@@ -0,0 +1,209 @@
+<!--
+  - @copyright Copyright (c) 2024 Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @author Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<NcModal v-if="modal"
+		:container="container"
+		@close="closeModal">
+		<div class="inbox">
+			<h2 class="inbox__heading">
+				{{ t('spreed', 'Pending invitations') }}
+			</h2>
+			<p class="inbox__disclaimer">
+				{{ t('spreed', 'Join conversations from remote Nextcloud servers') }}
+			</p>
+			<ul class="inbox__list">
+				<li v-for="item of invitations"
+					:key="`invitation_${item.id}`"
+					class="inbox__item">
+					<ConversationIcon :item="item" />
+					<div class="inbox__item-desc">
+						<span class="inbox__item-desc__name">
+							{{ item.roomName }}
+						</span>
+						<span class="inbox__item-desc__subname">
+							{{ t('spreed', 'From {user} at {remoteServer}', {
```

Comment:
missing user parameter

Reply:
We don't receive it from endpoint:
https://github.com/nextcloud/spreed/blob/b56912467344a829b6bafe7b1f91688d17c5b070/openapi-federation.json#L125

Reply:
yeah then we need to fix it, but merging with the placeholder is also weird :D

Reply:
Then I'll keep it as a draft until design feedback and API update is received